bin/MadGraph5_aMCatNLO/cards/production/13TeV/HeavyNeutrino_trilepton_V-0p01_tau_NLO/HeavyNeutrino_trilepton_mNgtmZ_V-0p01_tau_NLO/HeavyNeutrino_trilepton_V-0p01_tau_NLO_run_card.dat

```diff
@@ -58,10 +58,8 @@
 #***********************************************************************
 # PDF choice: this automatically fixes also alpha_s(MZ) and its evol.  *
 #***********************************************************************
-  lhapdf = pdlabel ! PDF set
-  $DEFAULT_PDF_SETS = lhaid                                           
-  $DEFAULT_PDF_MEMBERS  = reweight_PDF
-   
+  lhapdf	=  pdlabel    ! PDF set
+  292200	=  lhaid      ! if pdlabel=lhapdf, this is the lhapdf number
```

Comment:
How do you distinguish 2016 from 2017/2018 here. I don't see this indicated from the path name.

Reply:
We have different sets of cards and commits for 2016 and for 2017/2018. Sorry if this commit is confusing. In 2017/2018 we take the DEFAULT_PDF_MEMBERS and in 2016 we specify the lhaid